analysis/deciles_charts.py

```diff
@@ -129,10 +139,11 @@ def main():
 
     for measure_table in get_measure_tables(input_files):
         measure_table = drop_zero_denominator_rows(measure_table)
-        chart = get_deciles_chart(measure_table, config)
-        id_ = measure_table.attrs["id"]
-        fname = f"deciles_chart_{id_}.png"
-        write_deciles_chart(chart, output_dir / fname)
+        if config["charts"]["output"]:
```

Comment:
Right now if charts output is false then we do nothing? But in the future we might output the deciles table and just not the graphic, correct?

Reply:
That's correct. We may wish to revisit outputs, depending on whether we find it easy but confusing to output nothing. For the moment, though, this is a small step towards outputting tables, charts, tables and charts, or neither tables nor charts 🙂